gravitee-am-ui/src/app/domain/settings/theme/theme.component.ts

```diff
@@ -0,0 +1,127 @@
+/*
+ * Copyright (C) 2015 The Gravitee team (http://gravitee.io)
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *         http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+import {Component, OnChanges, OnInit, SimpleChanges} from '@angular/core';
+import {ActivatedRoute, Router} from '@angular/router';
+import {SnackbarService} from '../../../services/snackbar.service';
+import {AuthService} from '../../../services/auth.service';
+import {ThemeService} from "../../../services/theme.service";
```

Comment:
Neat: import should be single quote instead of double as per tslint config. Otherwise intellij shows error

Reply:
FP